doc(okd): add a section to deploy Longhorn by the manifest (#941)
longhorn/website: content/docs/1.7.0/advanced-resources/os-distro-specific/okd-support.md

  helm install longhorn longhorn/longhorn --namespace longhorn-system --create-namespace --set openshift.enabled=true
```

### Install With `oc` Command

> Install Longhorn on OKD Kubernetes cluster using this command:

Reviewer: ```suggestion
You can install Longhorn on [OKD](https://www.okd.io/) clusters using the following command:
```

Author: Modified. Thanks.

  ```shell
    oc apply -f https://raw.githubusercontent.com/longhorn/longhorn/v{{< current-version >}}/deploy/longhorn-okd.yaml
  ```

One way to monitor the progress of the installation is to watch pods being created in the `longhorn-system` namespace:

  ```shell
    oc get pods \
    --namespace longhorn-system \
    --watch
  ```

> See more details at [Install With Kubectl](../../../deploy/install/install-with-kubectl)

Reviewer: ```suggestion
For more information, see [Install with Kubectl](../../../deploy/install/install-with-kubectl).
```

Author: Modified. Thanks.
